src/components/Explorer/socialMeta.tsx

```diff
@@ -45,7 +45,7 @@ const getDescription = (journey: JourneyInfo) => {
 const getSocialMeta = (props: Props) => {
     const { journeyParams, journeys } = props;
     const enhanced = journeys && journeys[1];
-    const description = enhanced && getDescription(enhanced);
+    const description = enhanced && !"error" in enhanced && getDescription(enhanced);
```

Comment:
We can switch over to using `journey.navigationFailed` here too, right? I'd suggest putting a check for that inside of `getDescription`.

(At this point we have a `JourneyInfo` object and not a `JourneyApiRoute` so we can assume that `"error" in enhanced` is always false).